test/unit/request-body.test.js

```diff
@@ -79,6 +79,18 @@ describe('RequestBody', function () {
         });
     });
 
+    it('should work with string bodies correctly', function () {
+        var body = new RequestBody('foo');
+        expect(body).to.have.property('mode', 'raw');
+        expect(body).to.have.property('raw', 'foo');
+    });
+
+    it('should reject invalid body types', function () {
+        expect(new RequestBody([])).to.not.have.property('body');
```

Comment:
This will always pass 🤔

Reply:
Yes. The collection SDK didn't validate for non-standard body options before.

Reply:
why are we asserting `new RequestBody().body`? this will always be empty right? even for valid body types?

Reply:
Incorrect test; has been fixed.